src/services/cssHover.ts

```diff
@@ -108,6 +113,52 @@ export class CSSHover {
 			}
 		}
 
+		
+		if (hover) {
+			console.log(this.doesSupportMarkdown());
+			hover.contents = this.convertContents(hover.contents);
+		}
+
 		return hover;
 	}
+
+	private convertContents(contents: MarkupContent | MarkedString | MarkedString[]): MarkupContent | MarkedString | MarkedString[] {
+		if (!this.doesSupportMarkdown()) {
+			if (typeof contents === 'string') {
+				return contents;
+			}
+			// MarkupContent
+			else if ('kind' in contents) {
+				return {
```

Comment:
is kind is Markdown it could be returned as  MarkedString

Reply:
This is only when `!this.doesSupportMarkdown` is true

Reply:
MarkedString was part of the very first LSP, so in theory, it can always be returned

Reply:
If client doesn't support MD we should return `{ kind: 'plaintext' }` instead of `MarkedString` which would be interpreted as MD, don't we?